Content.Shared/Cloning/SharedCloningConsole.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Robust.Shared.Serialization;
+
+namespace Content.Shared.Cloning.CloningConsole
+{
+        [Serializable, NetSerializable]
+        public sealed class CloningConsoleBoundUserInterfaceState : BoundUserInterfaceState
+        {
+            public readonly string? ScannerBodyInfo;
+            public readonly string? ClonerBodyInfo;
+            // When this state was created.
+            // The reason this is used rather than a start time is because cloning can be interrupted.
+            public readonly TimeSpan ReferenceTime;
+            // Both of these are in seconds.
+            // They're not TimeSpans because of complicated reasons.
+            // CurTime of receipt is combined with Progress.
+            public readonly float Progress;
+            public readonly float Maximum;
```

Comment:
Pretty sure you're going to get prediction issues trying to track progress / pausing this way.